client/src/components/Profile/DoctorProfile.jsx

```diff
@@ -15,7 +16,36 @@ function DoctorProfile({
   isDoctorLoggedIn,
   phone,
   hospital,
+  handleOnHospitalChange,
+  handleOnPhoneChange,
 }) {
+  const [isPhoneOpen, setIsPhoneOpen] = useState(false);
+  const [isHospitalOpen, setIsHospitalOpen] = useState(false);
+  const [phoneError, setPhoneError] = useState("");
+
+  const handleOnHospitalClick = () => {
+    if (isHospitalOpen) {
+      handleProfileInfoChange("current_hospital", hospital);
+    }
+    setIsHospitalOpen(!isHospitalOpen);
+  };
+
+  const handleOnPhoneClick = () => {
+    if (phone.includes("-") || phone.includes("/") || phone.includes(" ")) {
+      phone = phone.replace(/[-/\s]/g, "");
+    }
```

Comment:
Nice. 

Another front end design option is to make it so that when they input their 10 digit number, split it into 3 text boxes.